### ISSUE TITLE
how to classify small-scale-image using resnet

### ISSUE THREAD

**Issue by scut-salmon** [keras-team/keras#11053]
Hi,
I would like to make a classification using resnet-50, however, the size of my image is only 64*64, which is smaller than the minimum size of (197, 197). How can I do it with my problem?
Thanks,
Zhang

**Comment by christyjacob4** [comment]
You will need to use some kind of zero padding to make your images match the dimensions. You can use the built-in zero padding layer from keras. 

keras.layers.ZeroPadding2D(padding=(1, 1), data_format=None)

check this for more information.
https://keras.io/layers/convolutional/#zeropadding2d
